src/components/title/index.tsx

```diff
@@ -1,4 +1,5 @@
 import styles from "./title.module.scss";
+import React from "react";
```

Comment:
NIT: Importing react is not necessary. In `Next JS` by default we have react imported

Reply:
Okay. Actually it is showing some error otherwise. Hence I imported react.

Reply:
@Khanjan04 After the new commit, does the code work properly? Because you said it was showing error if react was not imported.